pkg/operator/handler.go

```diff
@@ -348,10 +391,24 @@ func (h *Handler) Handle(ctx context.Context, event sdk.Event) error {
 				conditionResourceApply(o, operatorapi.ConditionFalse, fmt.Sprintf("unable to remove objects: %s", err), &statusChanged)
 			}
 		case operatorapi.Managed:
+			errOp := h.operatorStatus(osapi.OperatorProgressing, osapi.ConditionTrue, "registry server is in the process of deployment")
+			if errOp != nil {
+				logrus.Errorf("unable to update cluster status to %s=%s: %s", osapi.OperatorProgressing, osapi.ConditionFalse, errOp)
+			}
```

Comment:
@legionus I think this will need a follow up.  consider the case where we're doing a relist:  

we see the registryresource
we set the condition to progressing=true
we "apply changes" but there are no changes, so no deployment is triggered
we will not get a deployment update event to trigger us to set progressing=false (until the relist for deployments occurs).

So really we should not be setting "progressing=true" if we are not actually triggering a deployment.

again i'm going to mark this lgtm so we can move forward, but please plan to follow up on this (unless i've missed something).